libs/types/src/cdf/ballot-definition/index.test.ts

```diff
@@ -1,23 +1,44 @@
 import { buildSchema } from '@votingworks/cdf-schema-builder';
 import { readFileSync } from 'fs';
 import { join } from 'path';
+import { ok } from '@votingworks/basics';
 import { BallotDefinitionSchema } from '.';
 import { mockWritable } from '../../../test/helpers/mock_writable';
 import { testCdfBallotDefinition } from './fixtures';
+import {
+  isSubsetCdfSchema,
+  validateSchema,
+} from '../../../test/cdf_schema_utils';
+
+const nistXsd = readFileSync(
+  join(__dirname, '../../../data/cdf/ballot-definition/nist-schema.xsd'),
+  'utf-8'
+);
+const nistJson = readFileSync(
+  join(__dirname, '../../../data/cdf/ballot-definition/nist-schema.json'),
+  'utf-8'
+);
+const nistSchema = JSON.parse(nistJson);
+const vxJson = readFileSync(join(__dirname, './vx-schema.json'), 'utf-8');
+const vxSchema = JSON.parse(vxJson);
 
 test('BallotDefinition', () => {
   BallotDefinitionSchema.parse(testCdfBallotDefinition);
 });
 
 test('generated types are in sync with schema', () => {
-  const nistXsd = readFileSync(
-    join(__dirname, '../../../data/cdf/ballot-definition/nist-schema.xsd'),
-    'utf-8'
-  );
-  const vxJson = readFileSync(join(__dirname, './vx-schema.json'), 'utf-8');
   const generatedTypes = readFileSync(join(__dirname, './index.ts'), 'utf-8');
   const out = mockWritable();
   buildSchema(nistXsd, vxJson, out).unsafeUnwrap();
   const expectedTypes = out.toString();
   expect(generatedTypes).toEqual(expectedTypes);
 });
+
+test('VX schema is a valid JSON schema', () => {
```

Comment:
Looks like test is really "VX schema and NIST schema are valid JSON schemas", although we should hope the NIST schema is valid.

Reply:
Haha yeah I would hope! Updated the test title to make it less confusing: 2cff1610f